ibft/commit.go

```diff
@@ -77,34 +76,41 @@ func (i *Instance) uponCommitMsg() pipeline.Pipeline {
 					zap.String("Lambda", hex.EncodeToString(i.State.Lambda.Get())), zap.Uint64("round", i.State.Round.Get()),
 					zap.Int("got_votes", len(sigs)))
 
-				if aggMsg := i.aggregateMessages(sigs); aggMsg != nil {
-					i.decidedMsg = aggMsg
-					// mark instance commit
-					i.ProcessStageChange(proto.RoundState_Decided)
-					i.Stop()
+				aggMsg, err := i.aggregateMessages(sigs)
+				if err != nil {
+					i.Logger.Error("could not aggregate commit messages after quorum", zap.Error(err))
```

Comment:
shouldn't we break here if aggregation failed?

Reply:
In addition, round timer could be stopped while the aggregation fails
https://github.com/bloxapp/ssv/pull/285/files#diff-186423815922aeefaa19db97036033f00d2454e9dc88c19fbe33a1435ae5a5e6R73

In general the timer is being stopped by `instance.Stop()`, the call from this function to `stopRoundTimer` looks redundant

Reply:
We shouldn't stop if aggregation fails as you've decided.

not sure I understand the round timer comment